libbeat/idxmgmt/lifecycle/ilm_test.go

```diff
@@ -0,0 +1,229 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package lifecycle
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/elastic/beats/v7/libbeat/beat"
+)
+
+func TestDefaultSupport_Init(t *testing.T) {
+	info := beat.Info{Beat: "test", Version: "9.9.9"}
+
+	t.Run("with custom config", func(t *testing.T) {
+		tmp, err := DefaultSupport(nil, info, true)
+		require.NoError(t, err)
+
+		s := tmp.(*stdSupport)
+		assert := assert.New(t)
+		assert.Equal(true, s.Enabled())
+	})
+
+	t.Run("with custom alias config with fieldref", func(t *testing.T) {
+		tmp, err := DefaultSupport(nil, info, true)
+		require.NoError(t, err)
+
+		s := tmp.(*stdSupport)
+		assert := assert.New(t)
+		assert.Equal(true, s.Enabled())
+	})
```

Comment:
I'm not seeing any difference on those tests... Did you forget to add something in the second one?

Also you can use `assert.Equal` directly, which also avoids creating a variable shadowing the package name.
```suggestion
	t.Run("with custom config", func(t *testing.T) {
		tmp, err := DefaultSupport(nil, info, true)
		require.NoError(t, err)

		s := tmp.(*stdSupport)
		assert.Equal(t, true, s.Enabled())
	})

	t.Run("with custom alias config with fieldref", func(t *testing.T) {
		tmp, err := DefaultSupport(nil, info, true)
		require.NoError(t, err)

		s := tmp.(*stdSupport)
		assert.Equal(t, true, s.Enabled())
	})
```